types/schema/data/Applicant.ts

```diff
@@ -68,18 +64,21 @@ export interface Ownership {
    */
   ownersKnown?: 'all' | 'some' | 'none';
   /**
-   * @description Names and addresses of all known owners and agricultural tenants
+   * @description Names and addresses of all known owners and agricultural tenants, including date of notice or reason requisite notice has not been given, if applicable
    */
   owners?: {
+    interest?: 'owner' | 'tenant' | 'occupier' | 'other';
     name: string;
     address: Address | string;
+    noticeGiven?: boolean;
     noticeDate?: Date;
```

Comment:
Is this property still required?

> LDCs no longer report the `noticeDate` for individual owners

Reply:
Correct - it's actually a valid third scenario in the discriminated union above for PP & LBC apps :white_check_mark: